tests/test_utils/test_assigner.py

```diff
@@ -628,3 +628,15 @@ def test_mask_hungarian_match_assigner():
         dice_cost=dict(type='DiceCost', weight=0.0, pred_act=True, eps=1.0))
     with pytest.raises(AssertionError):
         self = MaskHungarianAssigner(**assigner_cfg)
+
+    # test with dicecost used in K-Net
+    assigner_cfg = dict(
```

Comment:
Please refer to https://github.com/open-mmlab/mmdetection/blob/3e2693151add9b5d6db99b944da020cba837266b/tests/test_utils/test_assigner.py#L598.
Put test for `naive_dice=True, naive_dice=False` together